index_matroska.md

```diff
@@ -187,7 +223,7 @@ In this case only the number of frames in the lace is saved, the size of each fr
 
 #### SimpleBlock Structure
 
-The SimpleBlock is very inspired by the Block structure. The main differences are the added Keyframe flag and Discardable flag. Otherwise everything is the same.
+The SimpleBlock is very inspired by the [Block structure](({{site.baseurl}}/index.html#block-structure). The main differences are the added Keyframe flag and Discardable flag. Otherwise everything is the same.
```

Comment:
We can skip out on the site.baseurl bit and just go with #block-structure, which works for the RFC.